scope.go

```diff
@@ -285,6 +285,8 @@ func (s *scope) killQuery() error {
 var allowedParams = []string{
 	"query",
 	"database",
+	"compress",
+	"decompress",
```

Comment:
Add comment with a link to clickhouse docs about `compress` and `decompress` params